test/jdk/javax/net/ssl/TLSv11/GenericBlockCipher.java

```diff
@@ -174,6 +175,7 @@ void doClientSide() throws Exception {
     public static void main(String[] args) throws Exception {
         // Re-enable TLSv1.1 since test depends on it.
         SecurityUtils.removeFromDisabledTlsAlgs("TLSv1.1");
+        SecurityUtils.removeFromDisabledTlsAlgs("TLS_RSA_*");
```

Comment:
You can put more than one alg in the same call, i.e. `SecurityUtils.removeFromDisabledTlsAlgs("TLSv1.1", "TLS_RSA_*");`

Reply:
Done